### PR TITLE
Adds new theme [tgcowell/waves]

### PR DESCRIPTION
**Pull request by tgcowell**

### CODE REVIEW

File: theme [hunk 1]
```diff
@@ -61,5 +61,6 @@
   "pbeckcom/green_slate_theme",
   "Poeschl/slate_red",
   "seangreen2/slate_theme",
-  "wowgamr/animated-weather-card"
+  "wowgamr/animated-weather-card",
+  "tgcowell/waves"
```
Comment (jbouwh) [inline theme:65]: Please keep the list sorted

**Review by jbouwh — CHANGES_REQUESTED** [review]
Please sort the list.

File: theme (hunk 1 above)
Comment (tgcowell) [reply theme]: Hi @jbouwh apologies I really should have noticed that before submitting! I have updated this as per your advice.

**Review by jbouwh — APPROVED** [review]

**Pull request merged**